src/store/courses/courses.test.js

```diff
@@ -0,0 +1,28 @@
+import { coursesReducer } from './courses';
+import { fetchAddCourse } from './coureses.actions';
+
+const initialState = {
+	courses: [],
+	courseForUpdate: {},
+};
+
+const newCourse = {
+	title: 'title',
+	description: 'description',
+	creationDate: '9/3/2021',
+	duration: 30,
+	authors: ['9b87e8b8-6ba5-40fc-a439-c4e30a373d36'],
+	id: '66cc289e-6de9-49b2-9ca7-8b4f409d6467',
+};
+
+describe('user', () => {
```

Comment:
typo "user"

Reply:
rename test to should contain userName and userName should be exist as this test depends on header component